project/themes/default/print.css

```diff
@@ -0,0 +1,31 @@
+html { height:0; }
```

Comment:
what you have changed in the print . css ?

Reply:
Did you test it on local machine ?

Reply:
I have changed the color for a better suits and tested already on local machine.

Reply:
1. I have merged your pull request 
2. In the future, before your create new pull request, please rebase your all your commit , which means merge all your previous commit to one commit , then new pull request

You can search the tutorial for rebase

The reason is that in the real practice in the IT company, in order to get clean code, each change should be applied to one commit instead of mutiple